cpu_scheduling_algorithms/round_robin_scheduling.cpp

```diff
@@ -0,0 +1,248 @@
+/**
+ * @file
+ * @brief Implementation of Round Robin CPU scheduling algorithm
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂